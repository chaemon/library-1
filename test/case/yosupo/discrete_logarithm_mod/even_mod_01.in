100
40280999 438515315 805540392
129099933 971852321 999893452
410134225 668461222 888826784
394300887 86086437 874944994
221329455 57893312 868221198
154204049 738660729 895021918
551666498 540440394 914338178
165573287 512342480 934107000
669622703 340240729 899635274
144902419 86035524 851717080
423893886 464748969 842271260
745991283 65304379 930638570
646596193 140145001 873791476
134706415 170189559 987083506
917006962 498351627 998984736
78228861 784928508 995873616
276287905 127534371 816670742
602636875 833497706 886981172
168745739 687931378 865762816
401296664 244718550 851112444
20801451 728508574 851139874
355924229 767675763 820219678
690017157 35094958 879540474
803294929 322191158 965296480
412368057 696252934 931690954
429649275 739609644 911871544
84017875 429336930 948926094
559811577 192014409 883902258
641105035 776026757 818419516
115129181 14794716 951131772
720571222 16609338 979173060
796633468 641040997 858381032
9273458 747543114 858462494
235611799 713122606 937674518
230441676 782661943 826491276
263420409 832590609 963984264
889752736 346283651 946325996
787563199 656787150 845387748
325699987 408388854 844235700
514686813 546683978 964012290
452720553 11844689 868234900
237462880 322160445 835743216
968998501 643437693 995187460
50514865 225072892 887365950
432384534 267319887 916023150
470850912 634063725 810785352
170224154 233237753 860530994
363661868 30383316 884172238
450267291 509435980 871094320
398793145 910647926 932859748
796833592 454223174 915984632
621479206 158107041 802669918
767759721 136585361 808232716
286340474 221735821 922835044
684185411 747165460 915169648
201499817 652532397 812603278
597836965 403721689 951765766
956715523 163056092 976494932
768003151 590039320 890466070
716350006 451084111 863550862
118564698 359276034 890090972
701887370 725160671 847409346
114167505 659739773 934088510
55285186 560410488 863848374
130521101 634454758 909953298
367621879 37543666 975208430
673784691 852411165 858778310
547390766 338428922 943646916
689572434 292588975 801722476
237458599 110596904 824529510
417951469 281530082 966123800
802111064 342905930 807469564
489807159 227640803 995350316
715231872 246388488 835825664
144892127 398434867 832200988
621850442 452366143 916717034
469546842 539193693 958253950
534692970 516992535 885102200
338125822 866018073 903980272
510544637 841323884 859517890
263447616 97154821 974114212
626682906 547354125 845942614
244996624 470490191 845876660
379219571 498680309 930661264
100482924 442510270 986841972
857344783 659400969 945720858
692474075 187583662 872844044
203919131 521450008 882113752
561050789 483834181 811159330
744939400 606001978 825601984
825332344 409340060 941547290
701010888 52772808 933014866
62382450 312651094 979236398
813796994 153189966 934383338
645498866 305360396 872983844
89137985 804282524 975910546
215551309 168847257 821098788
553423322 316656379 915056062
979254770 615808805 987506060
162058869 734380867 823287128
