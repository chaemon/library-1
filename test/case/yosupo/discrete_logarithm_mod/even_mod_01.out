-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
83476376
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
233294475
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
91390955
-1
-1
-1
-1
-1
